front/src/account/accessTokens/AccountNewAccessToken.tsx

```diff
@@ -19,8 +19,15 @@ export default function AccountNewAccessToken({
   return (
     <>
       <div className="notification success">
-        Un nouveau jeton d'accès a été généré ! Pensez à le copier maintenant.
-        Vous ne serez plus en mesure de le consulter ultérieurement.
+        <p>
+          {" "}
+          Un nouveau jeton d'accès a été généré ! Pensez à le copier maintenant.
+          Vous ne serez plus en mesure de le consulter ultérieurement.
+        </p>
+        <strong>
+          🚨 Ce jeton est confidentiel, ne le diffusez pas, et consultez nos
+          recommendations de sécurité dans la documentation.
```

Comment:
```suggestion
          recommandations de sécurité dans la documentation.
```

Peut-être ajouter un lien vers la doc ?